projects/L4T/packages/tegra-bsp/package.mk

```diff
@@ -1,177 +1,432 @@
+  rm -r lib
 
-  cp -PRv install/* ${INSTALL}/
-  cp -PRvn install/usr/lib/* ${TOOLCHAIN}/aarch64-libreelec-linux-gnueabi/sysroot/usr/lib/
+  # Same for sbin/
+  mv usr/sbin/* usr/bin/
+
+  #Symlink all files from nvidia locations to locations that libreELEC will link with.
+  #And skip symlinks
+  cd usr/lib/aarch64-linux-gnu/tegra
+    for filename in *; do
+      if [ -h "${filename}" ]; then
+        :
+        #rm -f ${filename}
+      else
+        ln -sf aarch64-linux-gnu/tegra/${filename} ../../${filename}
+      fi
+    done
+    cd ../tegra-egl
+    for filename in *; do
+      if [ -h "${filename}" ]; then
+        :
+        #rm  ${filename}
+      else
+        ln -sf aarch64-linux-gnu/tegra-egl/${filename} ../../${filename}
+      fi
+    done
+    cd ..
+    for filename in *; do
+      if [ -h "${filename}" ]; then
+        :
+        #rm  ${filename}
+      else
+        ln -sf aarch64-linux-gnu/${filename} ../${filename}
+      fi
+    done
+    cd ../../../
 
-  #install multimedia_api_headers
-  cp -PRvn multimedia_api/usr/src/jetson_multimedia_api/include/* ${SYSROOT_PREFIX}/usr/include
-  mkdir -p ${SYSROOT_PREFIX}/usr/src/jetson_multimedia_api/samples/common
-    cp -PRvn multimedia_api/usr/src/jetson_multimedia_api/samples/common/* ${SYSROOT_PREFIX}/usr/src/jetson_multimedia_api/samples/common/
+  # Remove unneeded files
+  rm -rf usr/lib/ld.so.conf usr/lib/ubiquity \
+	etc/{systemd,NetworkManager,fstab,lightdm,nv-oem-config.conf.t210,skel,wpa_supplicant.conf,enctune.conf,nv,nvphsd.conf,nvpmodel,xdg} \
+      usr/bin usr/sbin
+  rm -rf etc/systemd etc/sysctl.d etc/hostname etc/hosts etc/modprobe.d etc/modules-load.d var/nvidia usr/lib/systemd
+  rm -rf usr/lib/nvidia usr/share/backgrounds usr/share/doc usr/share/nvpmodel_indicator usr/share/polkit-1
+  rm -rf opt var
+
+  #remove not needed symlinks
+  rm usr/lib/libv4l usr/lib/tegra usr/lib/tegra-egl  usr/lib/nvidia.json
+
+  if [ "${DEVICE}" = "Switch" ]; then
+    #We dont need these with Switch UCM
+    rm -rf usr/share/alsa/init
+  fi
+
+  # Move udev from etc/ to usr/lib/
+  cp -PRv etc/udev usr/lib/
+  rm -rf etc/udev
+
+  # Refresh symlinks
+  cd usr/lib/
+  ln -sfn libcuda.so.1.1 libcuda.so
+  ln -sfn libdrm.so.2 libdrm_nvdc.so
+  ln -sfn libnvbufsurface.so.1.0.0 libnvbufsurface.so
+  ln -sfn libnvbufsurftransform.so.1.0.0 libnvbufsurftransform.so
+  ln -sfn libnvbuf_utils.so.1.0.0 libnvbuf_utils.so
+  ln -sfn libnvdsbufferpool.so.1.0.0 libnvdsbufferpool.so
+  ln -sfn libnvid_mapper.so.1.0.0 libnvid_mapper.so
+  ln -sfn libnvv4l2.so libv4l2.so.0.0.999999
+  ln -sfn libnvv4lconvert.so libv4lconvert.so.0.0.999999
+  ln -sfn libv4l2.so.0.0.999999 libv4l2.so.0
+  ln -sfn libv4l2.so.0 libv4l2.so
+  ln -sfn libv4lconvert.so.0.0.999999 libv4lconvert.so.0
+  ln -sfn libv4lconvert.so.0 libv4lconvert.so
+  ln -sfn libnvgbm.so libgbm.so.1
+  ln -sfn libnvidia-egl-wayland.so libnvidia-egl-wayland.so.1
+
+  if [ ! "${VULKAN}" = "" -o ! "${VULKAN}" = "no" ]; then
```

Comment:
say VULKAN=""

not("" equal "") or not("" = "no")
becomes
not(true) or not(false)
becomes
true or false
becomes
true

so I guess you want AND (-a) here (not empty and not no) - when using VULKAN.
or use VULKAN_SUPPORT yes / no. I do not think there will be any issue when upstreaming it if you make your point.